src/components/website-management/website-form-components/customization/BannerCustomization.js

```diff
@@ -0,0 +1,59 @@
+import {Button, Grid, TextField} from "@mui/material";
+import React, {useState} from "react";
+import CustomizationTabs from "./CustomizationTabs";
+import BannerTemplate from "./customization-components/BannerTemplate";
+import CookieBanner from "./customization-components/CookieBanner";
+import {Box} from "@mui/system";
+
+
+
+    const BannerCustomization = (props)=>{
```

Comment:
Let's left align the component code.

Reply:
Fixed the suggestion. commit no :- 747c6f3